docs/general/ddl/data-types.rst

```diff
@@ -561,6 +565,31 @@ Time zone syntax as defined by `ISO 8601 time zone designators`_.
     +-----------------------+
     SELECT 1 row in set (... sec)
 
+.. _date-data-type:
+
+``date``
+---------
+
+.. NOTE::
+
+    The storage of the ``date`` data type is not supported. Therefore,
+    it is not possible to create tables with ``date`` fields.
+
+The ``date`` data type consist of year, month, and day.
+A constant value of type ``date`` can be expressed by explicit casts 
+of string or numeric literals.
+
+The string format for dates is `YYYY-MM-DD`. For example `2021-03-09`.
```

Comment:
```suggestion
The ``date`` data type can be used to represent values with a year, month and a day.

To construct values of the type ``date`` you can cast a string literal or a numeric literal to ``date``.
If a numeric value is used, it must contain the number of milliseconds since ``1970-01-01T00:00:00Z``.

The string format for dates is `YYYY-MM-DD`. For example `2021-03-09`.
```